lib/XMLHttpRequest.js

```diff
@@ -37,7 +36,34 @@ XMLHttpRequest.XMLHttpRequest = XMLHttpRequest;
 function XMLHttpRequest(opts) {
   "use strict";
 
-  opts = opts || {};
+  // defines a list of default options to prevent parameters pollution
+  var default_options = {
+    pfx: void 0,
+    key: void 0,
+    passphrase: void 0,
+    cert: void 0,
+    ca: void 0,
+    ciphers: void 0,
```

Comment:
`undefined` would be just fine instead of `void 0`.

Reply:
sure i will change them to `undefined`